src/backup/BackupPhraseContainer.tsx

```diff
@@ -179,7 +179,7 @@ const styles = StyleSheet.create({
     flexWrap: 'wrap',
     flexDirection: 'row',
     marginTop: 8,
-    backgroundColor: colors.onboardingBackground,
+    backgroundColor: colors.ivory,
```

Comment:
Where is this backup phrase container used? Looking at the new designs for onboarding all of the backup phrase stuff I see looks like it has gray1 background ([example](https://www.figma.com/design/mthUlChSbWXXxLe2AV13Wd/Onboarding-2024?node-id=2512-4691&t=AT3fVvxFFHr78bl5-0))